modules/ROOT/pages/Development/BeginnersGuide/SimpleMod/gameworldmodule.adoc

```diff
@@ -56,28 +48,49 @@ These fields will be explained further when you prepare to upload your mod for o
 
 image:BeginnersGuide/simpleMod/EditPlugin.png[Edit your Plugin]
 
-== Create the Class
-
-It's time to create the InitGameWorld blueprint for our mod!
+Creating your plugin will create a plugin workspace folder under `<project>/Plugins/<Mod Reference>`.
 
 You might need to click on `View Options` -> `Show Plugins Content`
-in a Content Browser to view the contents of your mod.
+in a Content Browser to view the contents of your mod in the Unreal Editor.
 image:BeginnersGuide/simpleMod/ShowPluginContentInViewer.png[Show Plugin Content in Viewer]
 
 You can find your mod in the `<Your Mod Reference> Content`
 subfolder of the Content Browser root folder.
 
-This folder is where you will be putting most of the assets (models, blueprints, textures, etc.) that your mod uses.
+This folder is where you will be putting most of the assets
+(models, blueprints, textures, etc.) that your mod uses.
 
-[WARNING]
+[TIP]
 ====
-Be sure to read the notes on
-xref:Development/BeginnersGuide/index.adoc#_mod_reference[Mod References]
-if you have not yet done so.
-It is critical that your mod reference follows these format rules.
+If you want to Open Source your mod,
+or simply want to use a version control system like Git,
+this folder is the ideal place for you to create the repository.
+
+Using version control is essential for collaborating on a mod with others,
+it helps write patch notes when you release an update,
+and you can use it to go back to previous versions of your files if something breaks.
+
+If you work on multiple mods,
+you only need one full project setup
+(which can be the cloned SatisfactoryModLoader repository)
+and each mod that you make will be a plugin in this project setup.
+The plugin's folder would be the version control repository for that mod.
 ====
```

Comment:
I think this section points to the "Plugins/<Mod Reference>/Content"-Folder...
Tho the best place for adding source control should be "Plugins/<Mod Reference>"